script/generate_certs

```diff
@@ -11,7 +11,7 @@ $shell = ARGV.include? '-s'
 
 # Adapted from WEBrick::Utils. Skips cert extensions so it
 # can be used as a CA bundle
-def create_self_signed_cert(bits, cn, comment)
+def create_self_signed_cert(bits, cn)
```

Comment:
Change needed: Get rid of the `comment` passed in to this method in the usages (in this same file). I think 2 usages, maybe.